src/triangulum/utils.clj

```diff
@@ -63,9 +63,20 @@
 
 (def ^:private path-env (System/getenv "PATH"))
 
+(defn sh-exec-with
+  "Provides a given path and environment to one bash command,
+  and parses the output, creating an array as described in `parse-as-sh-cmd`.
```

Comment:
I think you meant "parses the input `command`", since there isn't any parsing of the outputs happening.

Reply:
Ah, yes you're right. I copied and pasted this part from `sh-wrapper` and forgot to update it.